libp2p/Host.h

```diff
@@ -87,7 +87,7 @@ struct Reputation
 class ReputationManager
 {
 public:
-    ReputationManager();
+    ReputationManager() = default;
```

Comment:
Is this needed at all?

Reply:
Ah you're right, C++ default constructors are declared & defined implicitly so we don't need this.